core/v2/types.go

```diff
@@ -309,7 +309,7 @@ func BatchFromProtobuf(proto *commonpb.Batch) (*Batch, error) {
 type Attestation struct {
 	*BatchHeader
 
-	// AttestedAt is the time the attestation was made
+	// AttestedAt is the time the attestation was made in nanoseconds
```

Comment:
Why not change the variable name itself to `AttestedAtNano`

Reply:
It's better to remain consistent with other timestamps in naming. I don't have objection if one wants to get all timestamps renamed by incorporating time units though.

Reply:
gotcha, agree its best to be consistent.